server/src/test/java/org/elasticsearch/plugins/PluginsUtilsTests.java

```diff
@@ -38,6 +37,21 @@
 @LuceneTestCase.SuppressFileSystems(value = "ExtrasFS")
 public class PluginsUtilsTests extends ESTestCase {
 
+    PluginDescriptor newTestDescriptor(String name, List<String> deps) {
```

Comment:
Maybe make deps variadic to avoid list creation by caller?

Reply:
We have to create a List for the descriptor regardless.